k8s/scripts/inspect.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+INSPECT_DUMP=$(pwd)/inspection-report
+
+SVC_ARGS_DIR=/var/snap/k8s/common/args
+K8SD_STATE_DIR=/var/snap/k8s/common/var/lib/k8sd/state
+K8SD_BIN=/snap/k8s/current/bin/k8sd
+SBOM_FILE=/snap/k8s/current/bom.json
+
+function log_success {
+    printf -- '\033[32m SUCCESS: \033[0m %s\n' "$1"
+}
+
+function log_info {
+	printf -- '\033[34m INFO: \033[0m %s\n' "$1"
+}
+
+function log_warning_red {
+	printf -- '\033[31m WARNING: \033[0m %s\n' "$1"
+}
```

Comment:
let's be consistent with indentation. also, please use 2 spaces, like we have in other places